django/template/loader_tags.py

```diff
@@ -315,13 +315,15 @@ def do_include(parser, token):
     while remaining_bits:
         option = remaining_bits.pop(0)
         if option in options:
-            raise TemplateSyntaxError('The %r option was specified more '
-                                      'than once.' % option)
+            raise TemplateSyntaxError(
+                'The %r option was specified more than once.' % option
+            )
         if option == 'with':
             value = token_kwargs(remaining_bits, parser, support_legacy=False)
             if not value:
-                raise TemplateSyntaxError('"with" in %r tag needs at least '
-                                          'one keyword argument.' % bits[0])
+                raise TemplateSyntaxError(
+                    '"with" in %r tag needs at least one keyword argument.' % bits[0]
```

Comment:
You may say this is out of scope. But thought it worth noting this is the first case I have spotted which has the string in single quotes and double quotes in the string. It seems elsewhere the string the user sees uses single quotes.

Reply:
In these cases `black` will leave quotes untouched. I don't think it's worth changing.